src/xcode/ENA/ENA/Resources/Localization/bg.lproj/Localizable.strings

```diff
@@ -451,6 +451,9 @@
 /* phone number; no spacing allowed */
 "ExposureSubmission_Hotline_Number" = "08007540002";
 
+/* phone number; no spacing allowed */
```

Comment:
As far as I remember we don't have to touch the Localizable strings in other languages. Translation Team take care of that.

Reply:
I asked Andreas and he told me that if I change in the German file something (any key change or addition), it will be paired into the other files and then translated. 
But because I renamed the keys, I wanted to ensure that this will be not forgotten. 
So my idea was: The translation team should see the german terms and will translate them for the correct key.

Reply:
Whatever we change in the english/turkish/... files will be overwritten by translation. For a developer test it might help  to have the text keys already in the localizable.strings files.